Formula/osx-ql.rb

```diff
@@ -0,0 +1,24 @@
+class OsxQl < Formula
+  desc "Opens files with the Quick Look feature"
+  homepage "https://github.com/morgant/tools-osx#ql"
+  url "https://github.com/morgant/tools-osx.git",
+      tag:      "ql-0.1",
+      revision: "0a2ec702c03670530c0eb799049c81daf648f414"
```

Comment:
we'd prefer tarball over git checkout

```suggestion
  url "https://github.com/morgant/tools-osx/archive/refs/tags/ql-0.1.tar.gz"
  sha256 "16c079be28651879325d7bec0863d61341752c2e6e1373e18cedc8e72d7c75eb"
```